### DISCUSSION TITLE
Edit User

### DISCUSSION THREAD

**Discussion by ddierschke** [Leantime/leantime#633] [General] [answered]
https://DOMAIN/users/editUser/1/ is not loading. The page is blank by all users.

**Comment by FrankSpadino** [comment]
Same problem here.

**Comment by jneug** [comment]
You can fix this by editing `src/domain/users/controllers/class.editUser.php` and adding an `>` on line 68. Change it to:

```php
'password' => ($row['password']),
```

**Reply by unknown797** [reply]
Thanks for the code, I had to reinstall it a few times thinking it was caused by my sub folder installation method :)

**Comment by FrankSpadino** [comment, marked as answer]
Thank you, it works fine!

**Comment by marcelfolaron** [comment]
This is now fixed in 2.1.9
